pkg/destroy/openstack/openstack.go

```diff
@@ -738,6 +789,8 @@ func deleteNetworks(opts *clientconfig.ClientOpts, filter Filter, logger logrus.
 			if !errors.As(err, &gerr) {
 				// This can fail when network is still in use
 				logger.Debugf("Deleting Network %q failed: %v", network.ID, err)
+				// Try to delete eventual leftover load balancers
+				deleteLeftoverLoadBalancers(opts, logger, network.ID)
```

Comment:
deleteLeftoverLoadBalancers returns bool and error. are you going to handle it?

Reply:
I wasn't planning to. I don't want to change the return value of `deleteNetworks()` based on the return of `deleteLeftoverLoadBalancers()`, since this would only be triggered when `deleteNetworks()` has failed already.

IMO, the destroy module needs a good refactoring that I don't want to commit to right now.

Reply:
I mean you created a new function `deleteLeftoverLoadBalancers` and it returns two parameters that you ignore. I think we need to either change the signature of the function, or handle the returned values.

Reply:
OK, I changed the signature. PTAL.